content/documentation/pages/1-installation/1-local/1-docker.md

```diff
@@ -0,0 +1,134 @@
+---
+path: 'installation/local/docker'
+title: 'Docker Compose'
+description: 'Installation using Docker Compose'
+---
+
+# Install using Docker Compose
+
+Spring Cloud Data Flow provides a Docker Compose file to let you quickly bring up Spring Cloud Data Flow, Skipper, Apache Kafka, Prometheus and pre-built dashboards for Grafana, instead of having to install them manually.
+Alternatively, you can follow the [manual installation steps](/documentation/master/installation/local/manual)
+
+[[tip | Upgrade to latest version of Docker ]]
+| We recommended that you upgrade to the [latest version](https://docs.docker.com/compose/install/) of Docker before running the `docker-compose` command. We have tested using Docker Engine version `18.09.2`
+
+## Download the Docker Compose File
+
+Download the Spring Cloud Data Flow Server Docker Compose file:
+
+```
+wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%dataflow-version%/spring-cloud-dataflow-server/docker-compose.yml
+```
+
+<!--NOTE-->
+
+If wget is unavailable, you can use curl or another platform-specific utility. [Click here](https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/%dataflow-version%/spring-cloud-dataflow-server/docker-compose.yml) to download the docker-compose.yml file.
+
+<!--END_NOTE-->
+
+The Docker Compose file will start up the following:
+
+1. Spring Cloud Data Flow Server
+1. Spring Cloud Skipper Server
+1. MySQL
+1. Apache Kafka
+1. Prometheus
+1. Grafana
+
+## Start Docker Compose
+
+In the directory where you downloaded `docker-compose.yml`, start the system, as follows:
+
+```
+export DATAFLOW_VERSION={local-server-image-tag}
```

Comment:
Can we use variables here so that the user can just do a copy paste?
Or remove them all together

Reply:
Yes, this will require the same logic mentioned in the previous comment.  I will change to use `%local-server-image-tag%` and review the entire doc in another pass for the asciidoc style usage of variables.